x-pack/plugins/uptime/e2e/journeys/utils.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Page } from '@elastic/synthetics';
+import { byTestId } from './uptime.journey';
+
+export async function waitForLoadingToFinish({ page }: { page: Page }) {
+  let isLoadingVisible = true;
+
+  while (isLoadingVisible) {
+    const loading = await page.$(byTestId('kbnLoadingMessage'));
+    isLoadingVisible = loading !== null;
+    await page.waitForTimeout(5 * 1000);
+  }
```

Comment:
I think we can simplify this function a little.

```suggestion
  while (true) {
    if ((await page.$(byTestId('kbnLoadingMessage'))) !== null) break; 
    await page.waitForTimeout(5 * 1000);
  }
```